### ISSUE TITLE
Exp bank

### ISSUE THREAD

**Issue by animenerdfreddurst** [animenerdfreddurst/ptu#1]
I need an experience bank on character (trainer) sheets, ideally alongside a window to apply it to pokemon during daily training)
Ideally, create a field that only GM can edit but is visible with a button beside it to apply exp. This would open a window with a dropdown of owned pokemon to apply exp with.
![image](https://github.com/animenerdfreddurst/ptu/assets/152929895/098952a7-f896-4779-bc81-4cbc70155103)

**Comment by npaisley** [comment]
Would you be adding the exp to teh player? And then are they going to allocate the EXP? This may be very difficult to implement due to the state of the code base

**Comment by animenerdfreddurst** [comment]
Yes. My thinking is you could have it as a button at the top of sheet accessible to GM only which opens a menu where a number of exp can be added to that player's bank. Then the Train Pokemon maneuver could trigger a separate menu for trainers that allows them to allocate exp from that bank to any of their pokemon.

**Comment by animenerdfreddurst** [comment]
Leaving this open until full implementation.
